site/learn/expert-advice/identity-basics/slow-migration.md

```diff
@@ -0,0 +1,252 @@
+---
+layout: advice
+title: Slow migration of user data
```

Comment:
Migration basics for legacy auth user data

Reply:
Unfortunately "legacy auth user data" doesn't make much sense. I will try to work "auth" in a bit more.